database-plugins/src/main/java/io/cdap/plugin/db/common/FQNGenerator.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.db.common;
+
+import io.opentracing.contrib.jdbc.ConnectionInfo;
+import io.opentracing.contrib.jdbc.parser.URLParser;
+
+/**
+ * Generate FQN from DB URL Connection
+ */
+
+public class FQNGenerator {
+  private static final String POSTGRES_DEFAULT_SCHEMA = "public";
```

Comment:
Do we already have a constant for it?

Reply:
I didn't find anything in DButils or Constants under hydrator-common